hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestMetaCache.java

```diff
@@ -253,6 +254,11 @@ public void testPreserveMetaCacheOnException() throws Exception {
           table.increment(increment);
           table.delete(delete);
           table.mutateRow(mutations);
+          // The value of the metaCacheHits counter is incremented by 6 in each round of the loop,
+          // for 0th iteration there will be 5 hits + 1 cache miss.
+          assertEquals(initialMetaCacheHits + 6 * i + 5, metrics.getMetaCacheHits());
+          // We will get a cache miss only on the first request, so the value will always be 1.
+          assertEquals(1, metrics.getMetaCacheMisses());
```

Comment:
Init `getMetaCacheMisses` too before asserting similar to `initialMetaCacheHits`? Thanks for the quick review addressal :)

Reply:
Would you mind taking a look on the newest commit and seeing if this is what you were expecting in your free time ? REALLY thanks for your patience ! :)